### DISCUSSION TITLE
How Do I setup proxies?

### DISCUSSION THREAD

**Discussion by Midnightzzz** [rdeepak2002/reddit-place-script-2022#389] [Q&A]
Any help would be appreciated

**Comment by Risewill23** [comment]
Read the readme

**Reply by Midnightzzz** [reply]
I have, doesnt work

**Reply by Midnightzzz** [reply]
> Read the readme

I have an error,  missing 1 required positional argument: 'proxies'

**Reply by thoroc** [reply]
Could you raise a bug then? Attach the output of the logs to it. It will be useful.
